src/vs/platform/externalTerminal/node/externalTerminalService.ts

```diff
@@ -96,7 +93,21 @@ export class WindowsExternalTerminalService extends ExternalTerminalService impl
 				windowsVerbatimArguments: true
 			};
 
-			const cmd = cp.spawn(WindowsExternalTerminalService.CMD, cmdArgs, options);
+			let spawnExec: string;
+			let cmdArgs: string[];
+
+			if (path.basename(exec) === 'wt') {
```

Comment:
and `wt.exe` maybe?

Reply:
Oh I meant to use `basename(exec, '.exe')`